recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/rcar-proprietary-graphic/rcar-proprietary-graphic.bb

```diff
@@ -0,0 +1,127 @@
+    install -m 644 ${S}/${includedir}/GLES3/*.h ${D}/${includedir}/GLES3/
+    install -d ${D}/${includedir}/KHR
+    install -m 644 ${S}/${includedir}/KHR/khrplatform.h ${D}/${includedir}/KHR/khrplatform.h
+
+    # Install pre-builded binaries
+    install -d ${D}/${libdir}
+    install -m 755 ${S}/${libdir}/*.so ${D}/${libdir}/
+    install -d ${D}/${exec_prefix}/local/bin
+    install -m 755 ${S}/${exec_prefix}/local/bin/dlcsrv_REL ${D}/${exec_prefix}/local/bin/dlcsrv_REL
+    install -d ${D}/lib/firmware
+    install -m 644 ${S}/lib/firmware/* ${D}/lib/firmware/
+
+    # Install pkgconfig
+    install -d ${D}/${libdir}/pkgconfig
+    install -m 644 ${S}/${libdir}/pkgconfig/*.pc ${D}/${libdir}/pkgconfig/
+
+    # Create symbolic link
+    cd ${D}/${libdir}
+    ln -s libEGL.so libEGL.so.1
+    ln -s libGLESv2.so libGLESv2.so.2
+
+    if [ "${USE_GLES_WAYLAND}" = "1" ]; then
+        # Set the "WindowSystem" parameter for wayland
+        if [ "${GLES}" = "gsx" ]; then
+            sed -i -e "s/WindowSystem=libpvrDRM_WSEGL.so/WindowSystem=libpvrWAYLAND_WSEGL.so/g" \
+                ${D}/${sysconfdir}/powervr.ini
+        fi
+    fi
+
+    # Install systemd service
+    if [ ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)} ]; then
+        install -d ${D}/${systemd_system_unitdir}/
+        install -m 644 ${S}/${systemd_system_unitdir}/rc.pvr.service ${D}/${systemd_system_unitdir}/
+        install -d ${D}/${exec_prefix}/bin
+        install -m 755 ${S}/${sysconfdir}/init.d/rc.pvr ${D}/${exec_prefix}/bin/pvrinit
+    fi
+
+    # Install kernel module gles
+    install -d ${D}/${sysconfdir}/modprobe.d
+    install -d ${D}/${sysconfdir}/modules-load.d
+    install -d ${D}/lib/modules/4.9.0-yocto-standard/extra
+    install -m 644 ${S}/lib/modules/4.9.0-yocto-standard/extra/pvrsrvkm.ko ${D}/lib/modules/4.9.0-yocto-standard/extra
+}
+
+PACKAGES = "\
+    ${PN} \
+"
+
+FILES_${PN} = " \
+    ${sysconfdir}/* \
+    ${libdir}/* \
+    /lib/modules/4.9.0-yocto-standard/extra/* \
+    /lib/firmware/rgx.fw* \
+    /usr/local/bin/* \
+    ${exec_prefix}/bin/* \
+    ${includedir}/* \
+    ${libdir}/pkgconfig/* \
+"
+
+PROVIDES = "virtual/libgles2 virtual/egl kernel-module-gles gles-user-module"
+RPROVIDES_${PN} += " \
+    kernel-module-pvrsrvkm \
+    kernel-module-dc-linuxfb \
+    gles-user-module \
+    ${GLES}-user-module \
+    libgles2-mesa \
+    libgles2-mesa-dev \
+    libgles2 \
+    libgles2-dev \
+"
+
+RDEPENDS_${PN} = " \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'libgbm wayland-kms', '', d)} \
+"
+
+INSANE_SKIP_${PN} = "ldflags build-deps file-rdeps"
+INSANE_SKIP_${PN}-dev = "ldflags build-deps file-rdeps"
+INSANE_SKIP_${PN} += "arch"
+INSANE_SKIP_${PN}-dev += "arch"
+INSANE_SKIP_${PN}-dbg = "arch"
+
+# Skip debug strip of do_populate_sysroot()
+INHIBIT_SYSROOT_STRIP = "1"
+
+# Skip debug split and strip of do_package()
+INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
+INHIBIT_PACKAGE_STRIP = "1"
```

Comment:
From what place these changes originally came from?

Reply:
From gles-user-module original recipe which is also install binary package